### PR TITLE
Feature/contact merge audit

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

This PR includes a variable to toggle on an audit adjustment for contacts that have been merged and removes them from the final queries.

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide explanation how the change is non breaking below.)

This PR only introduces a bug fix for users that were experiencing merged contacts persisting within the final models. This PR includes a variable to toggle on an audit adjustment for contacts that have been merged and removes them from the final queries.

**Is this PR in response to a previously created Issue**

- [X] Yes, Issue #46 
- [ ] No 

**How did you test the PR changes?** 

 
- [ ] CircleCi  
- [ ] Other (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 
- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
☎️ 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -60,14 +61,16 @@ vars:
...
+    hubspot_contact_merge_audit_enabled:  false
 
 models:
   hubspot:
     +materialized: table
     +schema: hubspot
     marketing:
       intermediate:
-        +materialized: ephemeral
+        +materialized: table
```
Comment (fivetran-joemarkiewicz) [inline dbt_project.yml:73]:
```suggestion
        +materialized: ephemeral
```

File: packages.yml [hunk 2]
```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/hubspot_source
-    version: [">=0.4.0", "<0.5.0"]
+  # - package: fivetran/hubspot_source
+  #   version: [">=0.4.0", "<0.5.0"]
+  - git: https://github.com/fivetran/dbt_hubspot_source.git
+    revision: feature/contact-merge-audit
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]: Swap before release

**Review by fivetran-jamie — APPROVED** [review]
lgtm

File: packages.yml (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply packages.yml:6]:
```suggestion
  - package: fivetran/hubspot_source
    version: [">=0.4.0", "<0.5.0"]
```

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/hubspot_source
-    version: [">=0.4.0", "<0.5.0"]
+  # - package: fivetran/hubspot_source
+  #   version: [">=0.4.0", "<0.5.0"]
+
+  - git: https://github.com/fivetran/dbt_hubspot_source.git
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]:
```suggestion
  - package: fivetran/hubspot_source
    version: [">=0.4.0", "<0.5.0"]
```

**Pull request merged**